src/scope.ts

```diff
@@ -65,7 +65,7 @@ export class Scope<T extends SpraypaintBase = SpraypaintBase> {
     return this._buildCollectionResult(response)
   }
 
-  async find(id: string | number): Promise<RecordProxy<T>> {
+  async find(id?: string | number): Promise<RecordProxy<T>> {
```

Comment:
I think the functionality of this whole PR is good, but as it's an uncommon case I don't love the type change of making `id` optional to find here, as for 99% of projects a missing id is incorrect and will result in confusing errors that the type system could easily catch. Perhaps we can come up with a parallel method that is only used for singular resources? `findOnly()` `load`, i'm not loving those but I'm open to other ideas.

Reply:
My idea was to let the singularResource variable determine the way the find() function behave. But i understand your point, i thought about making a parallel method too. does `get` suit you for the method name, or is it too generic ?

Reply:
what about `findSingle`?

Reply:
I'm agree with this name. I just made the change, can this be merge ?